doc/radosgw/multisite.rst

```diff
@@ -852,13 +852,13 @@ The CLI will echo a JSON object with the realm properties.
         "epoch": 1
     }
 
-Use ``>`` and an output file name to output the JSON object to a file.
+Use ``>`` and a file name to redirect the output of the JSON object to a file.
```

Comment:
Suggest  s/of the JSON object//
Though IMHO it's reasonable to expect someone to know how to redirect output, so really I'd take out the line altogether.

Reply:
That's an excellent pragmatic point. If you don't know about output redirection, you probably shouldn't be reading this documentation.

I'm removing the line entirely.